content/installation/administrator/admin_install_r/Rprofile.site

```diff
@@ -16,22 +15,65 @@ options(
   install.packages.check.source = "no",
   install.packages.compile.from.source = "never"
 )
-if (!"checklist" %in% rownames(utils::installed.packages())) {
-  utils::install.packages("checklist")
+if (
+  !all(
+    c("checklist", "fortunes", "remotes") %in%
+    rownames(utils::installed.packages())
+  )
+) {
+  if (Sys.getenv("RSTUDIO") == "1") {
+    warning(
+      "Missing packages. ",
+      "Please close RStudio and run R-",
+      paste(R.Version()[c("major", "minor")], collapse = "."),
+      "-win.exe first to install them automatically."
```

Comment:
I think you mean `Rgui.exe` below the R version specific path, not the Windows R installer file.

Reply:
fixed in 7c42478